app/javascript/components/upload/ExpandableFileForm.jsx

```diff
@@ -193,6 +198,23 @@ function DeleteButton({ file, deleteFile, setShowConfirmDeleteModal }) {
   return deleteButton
 }
 
+/**
+ * Determine whether to show the file upload control buttons.
+ * If the file form is for Clustering, Expression, or
+ * Metadata while in AnnData experience mode do not display the buttons
+ *
+ * @param {boolean} isAnnDataExperience
+ * @param {array} allFiles
+ * @returns {boolean} Whether to show upload control buttons or not
+ */
+function displayFileUploadControls(isAnnDataExperience, allFiles) {
```

Comment:
Blocker: the name of this function indicates it _displays_ file upload controls, which is confusing as it doesn't do that, and instead returns whether file upload controls _should be displayed_.  Refining this seems important for maintainability.

For functions that return Booleans, SCP JS convention is to name it something like `getFooEnabled` (e.g. `getShowHideEnabled`) or `getIsBaz` (e.g. `getIsRefGroup`).  

```suggestion
function getIsUploadEnabled(isAnnDataExperience, allFiles) {
```